File: Axiom/Internal/Ast/ExpressionStatement.cs
namespace Axiom.Internal.Ast
{
    internal class ExpressionStatement : Statement
    {
        public ExpressionStatement(Expression expression)
        {
            Expression = expression;
        }

        public override void Accept(IVisitor visitor)
        {
            Expression.Accept(visitor);
        }

        public Expression Expression { get; set; }
    }
}

File: src/main/java/com/moybl/axiom/ast/ExpressionStatement.java
package com.moybl.axiom.ast;

public class ExpressionStatement implements Statement {

	private Expression expression;

	public ExpressionStatement(Expression expression) {
		this.expression = expression;
	}

	public void accept(Visitor visitor) {
		expression.accept(visitor);
	}

	public Expression getExpression() {
		return expression;
	}

}
